### PR TITLE
fix(components/Slider): align icons with slider handler and add disabled props

### PR DESCRIPTION
**Pull request by nicomaligne**
**What is the problem this PR is trying to solve?**
We need to align the icons with the button value of the slider.
rating25
You can see the guideline here : http://guidelines.talend.com/document/92132#/specific-features/rating-sliders
**What is the chosen solution to this problem?**
Tweak some css and html
We also add a disabled props that allow to disabled the buttons and the slider.
**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/stories/Slider.js [hunk 1]
```diff
@@ -48,52 +48,56 @@ const actions = [
...
+		icon: 'talend-smiley-satisfied',
+		'data-feature': 'action',
+		link: true,
+		hideLabel: true,
+	},
 ];
 
 const functionToFormat = value => `${value}-test`;
-const functionFormatFloor = value => Math.floor(value);
+const functionFormatFloor = value => `${Math.floor(value)}k`;
```
Comment (jare-talend) [inline packages/components/stories/Slider.js]: what k ?

File: packages/components/src/Slider/Slider.component.js [hunk 2]
```diff
@@ -219,17 +224,19 @@ class Slider extends React.Component {
...
-					id={id}
-					value={value}
-					min={min}
-					max={max}
-					handle={noValue ? undefined : this.state.handle}
-					className={classnames(theme['tc-slider-rc-slider'], 'tc-slider-rc-slider')}
-					onChange={onChange}
-					{...rest}
-				/>
+			<div>
```
Comment (jare-talend) [inline packages/components/src/Slider/Slider.component.js]: why double div nesting ? one only

File: packages/components/stories/Slider.js (hunk 1 above)
Comment (nicomaligne) [reply packages/components/stories/Slider.js]: It was just to look like the visual on frontify. I can remove it.

File: packages/components/src/Slider/Slider.component.js (hunk 2 above)
Comment (nicomaligne) [reply packages/components/src/Slider/Slider.component.js]: Need the slider padding applied only to the slider not the icons.

**Pull request merged**